bootstrap/test/basis/file/test_file_full_sq.expected

```diff
@@ -0,0 +1,3 @@
de": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}, {"flags": 0, "ioprio": 0, "fd": 0, "opcode": "IORING_OP_NOP", "user_data": {"refcount": 1, "opcode": "IORING_OP_NOP", "cqe": {"res": 0, "flags": 0, "user_data": "NULL"}}}]}
```

Comment:
😢 Might as well just print a hash value.

Reply:
I'm going to think about this a little. We have a convenient string hash in our Basis library, but it might take a bit of effort (or ugly hackery) to get this output into OCaml. Do you think this is spammy enough that we need to go through the effort?